app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -74,16 +75,19 @@ class NewMessageViewModel : ViewModel() {
     }
 
     private fun configureDraft(draftUuid: String? = null) = viewModelScope.launch(Dispatchers.IO) {
```

Comment:
Configure draft ? 🤔 It's just to create a draft or init values, right?

Reply:
Will be fixed in #314.